index.html

```diff
@@ -4,9 +4,139 @@
+      <p class="headline-header primary-font">"Hello! Sharing world"</p>
+      <p class="headline-body secondary-font">CREATIVE COMMONS GLOBAL SUMMIT 2015</p>
+    </div>
+    <div class="headline-body-intro">
+      <p class="body-intro primary-font">A joyful summit where we believe in the value of openness and sharing to create a positive change with people from over 80 countries is taking place in Octber, in Nigeria</p>
+    </div>
+    
+    <div class="time secondary-font">2015.10.15(THU) ~ 16(FRI)</div>
+    <P class="venue primary-font">@ National Museum of Nigeria, Art Center Lagos and more</P>
+  </header>
+  <section class="main-program-section">
+    <div class="overlay">
+    <h1 class="program-head primary-font">
+      Main Program
+    </h1>
+    <div class="program-section">
+     
+      <div class="card-container">
+        <img src="./assets/images/Lecture.svg" alt="lucture logo" class="card-img">
+        <h5 class=" card-title primary-font">Lecture</h5>
+        <p class="card-text primary-font">Listen to the speakers from various countries about the messages of sharing and opening new ideas.</p>
+      </div>
+     <div class="card-container">
+        <img src="./assets/images/Exhibition.svg" alt="Exhibition logo" class="card-img">
+        <h5 class="card-title primary-font">CC Exhibition</h5>
+        <p class="card-text primary-font">Appreciate various creations applying CC license of artists, organized from Art Center Lagos</p>
+      </div>
+       <div class="card-container">
+        <img src="./assets/images/Forum.svg" alt="forum logo" class="card-img">
+        <h5 class="card-title primary-font">Forum</h5>
+        <p class="card-text primary-font">Have the time to share your thoughts and opinions for each topic.</p>
+      </div>
+      <div class="card-container">
+        <img src="./assets/images/Workshop.svg" alt="workshop logo" class="card-img">
+        <h5 class="card-title primary-font">Workshop</h5>
+        <p class="card-text primary-font">Try creating your own work using open source license rather than just watching at it.</p>
+      </div>
+      <div class="card-container">
+        <img src="./assets/images/ignite.svg" alt="ignite logo" class="card-img">
+        <h5 class="card-title primary-font">CC Ignite</h5>
+        <p class="card-text primary-font">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+      </div>
+    </div>
+    <a href="#" class="link primary-font">Join CC Global Submit 2015</a>
+    <a href="" class=" fullProgram tetiary-font">SEE THE WHOLE PROGRAM</a>
+
+  </div>
+  </section>
+  <h3 class="featured primary-font"> Featured speakers</h3>
+  <section class="speaker-section" >
+    <div class="speaker-details " id="speaker-details">
+      
+    </div>  
+    
+  </section>
+  <div class="down-btn">
+
+    <button type="button" class="more-future primary-font" id="showMore"><p class="more-btn">More</p>  
```

Comment:
- [ OPTIONAL ] Great job adding the functionality of the `More` button. However, You didn't add the right styles to it as shown in the design so kindly add this as well to improve the overall look and feel of your project :+1: 
- Your Design 
 ![image](https://user-images.githubusercontent.com/102699958/227722470-2f4ca235-4b44-4603-b3b8-916080768293.png)
- The Original Design
 ![image](https://user-images.githubusercontent.com/102699958/227722507-819c3339-4e1c-49b1-a1f0-0c1984b96017.png) |